// Copyright (c) 2023 Boston Dynamics AI Institute LLC. All rights reserved.

#include <rclcpp/node_options.hpp>
#include <rclcpp_components/register_node_macro.hpp>
#include <spot_driver/images/spot_image_publisher_node.hpp>

RCLCPP_COMPONENTS_REGISTER_NODE(spot_ros2::images::SpotImagePublisherNode)
